client/lib/wp/sync-handler/utils.js

```diff
@@ -1,11 +1,10 @@
 /**
  * External dependencies
  */
-import sha1 from 'js-sha1';
+import sha1 from 'hash.js/lib/hash/sha/1';
```

Comment:
in fact, we could get rid of sync-handler as well. In this way, we could clean this module completely.